ROMFS/px4fmu_common/init.d/13009_vtol_spt_ranger

```diff
@@ -75,7 +75,7 @@ then
 	param set PWM_AUX_REV3    1
 	param set PWM_AUX_REV4    1
 
-	param set PWM_AUX_DIS5 950
+	param set PWM_AUX_DISARMED 950
 fi
```

Comment:
Again, similar to 13000_generic_vtol_standard, this file was originally:
```
set PWM_ACHDIS 5
set PWM_AUX_DISARMED 950
```
and will require the same corrective action as 13000_generic_vtol_standard.